src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PurposesApiServiceImpl.scala

```diff
@@ -355,14 +355,14 @@ final case class PurposesApiServiceImpl(
     toEntityMarshallerProblem: ToEntityMarshaller[Problem],
     toEntityMarshallerCreatedResource: ToEntityMarshaller[CreatedResource]
   ): Route = {
-    logger.info(s"Creating purpose with seed $purposeSeed")
+    logger.info(s"Creating purpose")
```

Comment:
thank you for removing the seed! 
May I ask you to add to the log the `eserviceId` and the `consumerId` to help identify the purpose in case of issues?
If we log specific fields that we know to be safe there are no security issues